### DISCUSSION TITLE
Enable the position of arrows to be customized

### DISCUSSION THREAD

**Discussion by mcmxcdev** [davidjerleke/embla-carousel#473] [Ideas]
We have a product requirement to have the carousel wrap cards, but with different positioning of arrows:

![image](https://github.com/davidjerleke/embla-carousel/assets/16797721/35b6bf7c-f5ef-484f-b366-5c086e69dde6)

Would it be possible to give the position of arrows flexibility in a way that they can be positioned next to the dots?

**Comment by davidjerleke** [comment]
Hi @mcmxcdev,

Thanks for your question. Arrows and dots aren’t something totally  Embla generates for you. Embla provides API methods so you can setup them yourself. That means that you can place the dots and arrows wherever you want, as long as you have access to the `emblaApi` instance.

1. Go to the [arrows and dots example](https://www.embla-carousel.com/examples/static/#arrows--dots).
2. Click on “Edit code”.
3. Choose setup.
4. Click on “Create Sandbox”.
5. Modify the sandbox to your liking.

Best,
David
